src/extension.ts

```diff
@@ -508,6 +517,34 @@ class Device extends vscode.TreeItem {
             [this.shellPort.toString()]);
         term.show();
     }
+    
+    async captureScreenshot() {
+        const error = (message) => {
+            vscode.window.showErrorMessage("Error capturing screenshot: " + message);
+        }
+        
+        try {
+            const screenshotDirectory = await getSharedTempDir('ev3dev-screenshots');
+            const screenshotFile = `${screenshotDirectory}/ev3dev-${sanitizedDateString()}.png`;
+
+            const conn = await this.exec('fbgrab -');
+            const writeStream = fs.createWriteStream(screenshotFile);
+            writeStream.on('open', () => {
+                conn.stdout.pipe(writeStream);
+            });
+
+            writeStream.on('close', () => {
+                vscode.window.setStatusBarMessage(`Screenshot successfully captured`);
+                vscode.commands.executeCommand('vscode.open', vscode.Uri.file(screenshotFile));
+
+                // Delay to allow viewer to open
+                setTimeout(() => vscode.commands.executeCommand('workbench.action.files.saveAs'), 100);

```

Comment:
Hmmm... delays like this are never good. If someone is on a really slow computer it won't work right.

I'm thinking maybe we just display the file and users can click File > Save As... if they want to save it somewhere else.

I also found that I can click and drag the image to other programs, so I probably won't use the File > Save As... myself, so it would just be getting in my way for me.